_posts/2017-02-02-Rust-1.15.md

```diff
@@ -0,0 +1,288 @@
+---
+layout: post
+title: "Announcing Rust 1.15"
+author: The Rust Core Team
+---
+
+The Rust team is happy to announce the latest version of Rust, 1.15.0. Rust is a
+systems programming language focused on safety, speed, and concurrency.
+
+If you have a previous version of Rust installed, getting Rust 1.15 is as easy as:
+
+```bash
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`][install] from the
+appropriate page on our website, and check out the [detailed release notes for
+1.15.0][notes] on GitHub. 1443 patches were landed in this release.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1150-2017-02-02
+
+### What's in 1.15.0 stable
+
+Rust 1.15 sees an _extremely_ long-awaited feature land on stable: custom
+derive! To review, in Rust, you've always been able to automatically implement
+a trait through the `derive` attribute:
+
+```rust
+#[derive(Debug)]
+struct Pet {
+    name: String,
+}
+```
+
+The `Debug` trait is then implemented for `Pet`. However, this only worked for
```

Comment:
Maybe add a line like "this vastly reduces the boilerplate code you need to write" to motivate using `derive`